pkg/controller/utils/infinispan/configuration/xsite.go

```diff
@@ -0,0 +1,289 @@
+package configuration
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+
+	"github.com/go-logr/logr"
+	ispnv1 "github.com/infinispan/infinispan-operator/pkg/apis/infinispan/v1"
+	consts "github.com/infinispan/infinispan-operator/pkg/controller/constants"
+	"github.com/infinispan/infinispan-operator/pkg/controller/infinispan/util"
+	"github.com/infinispan/infinispan-operator/pkg/controller/utils/common"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	restclient "k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// CreateInfinispanConfiguration generates a server configuration
+func (xsite *XSite) CreateInfinispanConfiguration(name string, loggingCategories map[string]string, namespace string) InfinispanConfiguration {
```

Comment:
Is this func strictly related to xsite? It seems a general Infinispan function

Reply:
Oh, yes, it was originally in configuration.go and suddenly moved, fixed